openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1521,7 +1537,7 @@ def populate_load_placeholder(self, placeholder, ignore_repre_ids=None):
             return
 
         repre_load_contexts = get_contexts_for_repre_docs(
-            self.project_name, filtered_representations
+            self.current_project_name, filtered_representations
```

Comment:
Tested with following result:
```
Traceback (most recent call last):
  File "C:\CODE\__PYPE\OpenPype_3\openpype\pipeline\workfile\workfile_template_builder.py", line 679, in populate_scene_placeholders
    placeholder_plugin.populate_placeholder(placeholder)
  File "C:\CODE\__PYPE\OpenPype_3\openpype\hosts\nuke\api\workfile_template_builder.py", line 183, in populate_placeholder
    self.populate_load_placeholder(placeholder)
  File "C:\CODE\__PYPE\OpenPype_3\openpype\pipeline\workfile\workfile_template_builder.py", line 1540, in populate_load_placeholder
    self.current_project_name, filtered_representations
  File "C:\CODE\__PYPE\OpenPype_3\openpype\pipeline\load\utils.py", line 163, in get_contexts_for_repre_docs
    version_doc = version_docs_by_id[ repre_doc["parent"] ]
    ```
    
    I was checking the data in DB and id on parent is available. And the project was in DB.

Reply:
I was printing result of the `self.current_project_name` variable and it was pushing the actual project rather than the Library project.